src/controllers/accomodation.controller.js

```diff
@@ -323,14 +323,18 @@ class Accommodation {
     const accommodations = await accomodationServices.findAccomodationByCity(destinaton);
     const accommodationDetails = [];
     await Promise.all(accommodations.map(async (accomodation) => {
```

Comment:
@Jajabenit250 instead of creating an additional array `accommodationDetails `, can't you get array that contains accommodation details from accommodations.map?

Reply:
@dushi I know that. the reason why I used accommodationsDetails is that  i only need

>  id, name, availableRooms, averageRate, firstImage
in a bunch of many information that why I used that array to only capture those pieces of information..